client/app/src/main/java/com/microsoft/research/karya/ui/dashboard/DashboardFragment.kt

```diff
@@ -58,16 +59,23 @@ class DashboardFragment : Fragment(R.layout.fragment_dashboard) {
   private fun observeUi() {
     viewModel.dashboardUiState.observe(lifecycle, lifecycleScope) { dashboardUiState ->
       when (dashboardUiState) {
-        is DashboardUiState.Success -> showSuccessUi(dashboardUiState.data)
+        is DashboardUiState.Success -> showSuccessUi(dashboardUiState.taskInfoData, dashboardUiState.totalCreditsEarned)
         is DashboardUiState.Error -> showErrorUi(dashboardUiState.throwable)
         DashboardUiState.Loading -> showLoadingUi()
       }
     }
   }
 
-  private fun showSuccessUi(taskInfoList: List<TaskInfo>) {
+  private fun showSuccessUi(taskInfoList: List<TaskInfo>, totalCreditsEarned: Float?) {
     hideLoading()
     (binding.tasksRv.adapter as TaskListAdapter).updateList(taskInfoList)
+    // Show total credits if it is greater than 0
+    if (totalCreditsEarned != null && totalCreditsEarned > 0) {
+      binding.rupeesEarnedCl.visible()
+      rupeesEarnedTv.text = "%.2f".format(totalCreditsEarned)
+    } else {
+      binding.rupeesEarnedCl.gone()
+    }
```

Comment:
This should be done in the error state too.

Reply:
Why you wanna hide total credits earned so far in error state?